assets/js/app.js

```diff
@@ -25,3 +25,7 @@ import './header';
 import './filter_players';
 import './filter_trade_form';
 import './confirm_submit';
+
+import StandingsChart from './standings_chart.js'
+
+StandingsChart.buildChart()
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)